tanner/utils/mysql_db_helper.py

```diff
@@ -15,15 +15,23 @@ def __init__(self):
         self.logger = logging.getLogger('tanner.db_helper.MySQLDBHelper')
 
     async def connect_to_db(self):
+        """
+        Creates a aiomysql connection
+        :return: connection object
+        """
+
         conn = await aiomysql.connect(host=TannerConfig.get('SQLI', 'host'),
                                       user=TannerConfig.get('SQLI', 'user'),
                                       password=TannerConfig.get('SQLI', 'password')
                                       )
         return conn
 
     async def check_db_exists(self, db_name, ):
```

Comment:
could you please remove additional comma in function parameters? :) I don't know why it is here :D

Reply:
Maybe its mainly for multiline code? 
https://github.com/mushorg/tanner/pull/315#discussion_r287697507

Reply:
I would say it's a typo

Reply:
Okay.